build.rs

```diff
@@ -115,11 +147,14 @@ fn main() {
         }
     }
     c_build.file("src/util/util.c");
+
```

Comment:
Extraneous blank line?

Reply:
It was intentional, as it splits configuring the C build from actually doing it. Can kill it if you like.